soundbay/inference.py

```diff
@@ -134,8 +134,45 @@ def inference_to_file(
     output_file = output_path / filename
     concat_dataset.to_csv(index=False, path_or_buf=output_file)
 
+    # save raven file
+    threshold = 0.5
+    thresholdtext = int(threshold*10)
+    raven_filename = f"raven_annotations-{datetime.datetime.now().strftime('%Y-%m-%d_%H-%M-%S')}-{model_name}-{dataset_name}-thresh0{thresholdtext}.csv"
+    raven_output_file = output_path / raven_filename
+    raven_df = inference_csv_to_raven(results_df, test_dataset, threshold=threshold)
+    raven_df.to_csv(index=False, path_or_buf=raven_output_file, sep="\t")
+
     return
 
+def inference_csv_to_raven(probsdataframe, test_dataset, threshold=0.5):
+    # transforms the probability dataframe to a raven format with class_1 predictions as the annotated bounding boxes
+    len_dataset = probsdataframe.shape[0] #number of segments in wav
+    seq_len = test_dataset.seq_length
```

Comment:
if the whole use of `test_dataset` is its attribute `seq_length`, I think `seq_length` should be the function argument and not `test_dataset`